Source/Core/DolphinWX/Debugger/BreakpointWindow.cpp

```diff
@@ -57,33 +49,62 @@ class CBreakPointBar : public DolphinAuiToolBar
     AddTool(ID_ADDMC, "+MBP", m_Bitmaps[Toolbar_Add_MC]);
     Bind(wxEVT_TOOL, &CBreakPointWindow::OnAddMemoryCheck, parent, ID_ADDMC);
 
-    AddTool(ID_LOAD, _("Load"), m_Bitmaps[Toolbar_Delete]);
+    AddTool(ID_LOAD, _("Load"), m_Bitmaps[Toolbar_Load]);
     Bind(wxEVT_TOOL, &CBreakPointWindow::Event_LoadAll, parent, ID_LOAD);
 
-    AddTool(ID_SAVE, _("Save"), m_Bitmaps[Toolbar_Delete]);
+    AddTool(ID_SAVE, _("Save"), m_Bitmaps[Toolbar_Save]);
     Bind(wxEVT_TOOL, &CBreakPointWindow::Event_SaveAll, parent, ID_SAVE);
   }
 
+  void ReloadBitmaps()
+  {
+    Freeze();
+
+    InitialiseThemedBitmaps();
+    for (int i = 0; i < ID_NUM; ++i)
+      SetToolBitmap(i, m_Bitmaps[i]);
+
+    Thaw();
+  }
+
 private:
   enum
   {
-    Toolbar_Delete,
+    Toolbar_Delete = 0,
+    Toolbar_Clear,
     Toolbar_Add_BP,
     Toolbar_Add_MC,
+    Toolbar_Load,
+    Toolbar_Save,
     Num_Bitmaps
   };
 
   enum
   {
-    ID_DELETE = 2000,
+    ID_DELETE = 0,
     ID_CLEAR,
     ID_ADDBP,
     ID_ADDMC,
     ID_LOAD,
-    ID_SAVE
+    ID_SAVE,
+    ID_NUM
   };
 
+  void InitialiseThemedBitmaps()
+  {
+    wxSize bitmap_size = FromDIP(wxSize(24, 24));
+    SetToolBitmapSize(bitmap_size);
+
+    for (std::size_t i = 0; i < m_image_names.size(); ++i)
+      m_Bitmaps[i] =
+          WxUtils::LoadScaledThemeBitmap(m_image_names[i], this, bitmap_size, wxDefaultSize,
+                                         WxUtils::LSI_SCALE_DOWN | WxUtils::LSI_ALIGN_CENTER);
+  }
+
   wxBitmap m_Bitmaps[Num_Bitmaps];
+  const std::array<const char* const, Num_Bitmaps> m_image_names{
```

Comment:
This should likely be `static const`

Reply:
moved this and the other one to the only function it's used as I realised I didn't needed this to be class wide.